pkg/otlp/internal/serializerexporter/config.go

```diff
@@ -165,18 +155,3 @@ type metricsExporterConfig struct {
 func (e *exporterConfig) Validate() error {
 	return e.QueueSettings.Validate()
 }
-
-func (e *exporterConfig) Unmarshal(cfgMap *config.Map) error {
```

Comment:
If `Unmarshal` is not defined the Collector will use the default unmarshaling function (basically, calling `UnmarshalExact`), so, since we don't handle deprecations right now, this can be removed.